server/service/integration_mdm_test.go

```diff
@@ -12840,7 +12843,7 @@ INSERT INTO host_mdm_apple_declarations (
 		for _, m := range r.Declarations.Configurations {
 			// look up the declaration by the server token (we trim the token to the first seven
 			// chars to match our keys because response is padded to length 16 with "\u000")
```

Comment:
We can probably delete the above comment now if we don't need to slice to the first seven?

Reply:
good catch, thanks!